x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/store/middleware.ts

```diff
@@ -369,7 +368,7 @@ export const endpointMiddlewareFactory: ImmutableMiddlewareFactory<EndpointState
     if (
       action.type === 'userChangedUrl' &&
       hasSelectedEndpoint(getState()) === true &&
-      getEndpointDetailsFlyoutView(getState()) === EndpointDetailsTabsTypes.activityLog
+      uiQueryParams(getState()).show === EndpointDetailsTabsTypes.activityLog
```

Comment:
I see we do this check in a few places - maybe create a dedicated selector for it? like `getIsOnEndpointDetailsActivityLog(state): boolean`